src/ansys/fluent/core/launcher/pim_launcher.py

```diff
@@ -229,7 +229,7 @@ def launch_remote_fluent(
         product_name=(
             "fluent-meshing"
             if FluentMode.is_meshing(mode)
-            else "fluent-2ddp" if dimensionality == "2d" else "fluent-3ddp"
+            else "fluent-2ddp" if dimensionality == Dimension.TWO else "fluent-3ddp"
```

Comment:
I think we can also create an issue for future work to factor out code duplication from these launchers.

Reply:
@seanpearsonuk Can you please point out to code duplication in these launchers.

Reply:
Some logic for handling arguments (setting to a chosen default where the argument value is None) appears in more than one place. There is opportunity for consolidation there. I also note that some arguments are unused and I'm not sure if that's intentional in some way.

Reply:
@seanpearsonuk Ok. Thanks a lot.